src/CommandExecutor.ts

```diff
@@ -76,14 +118,17 @@ class NullExecutionStrategy extends CommandExecutionStrategy {
 export default class CommandExecutor {
     private static executors = [
         BuiltInCommandExecutionStrategy,
-        SystemFileExecutionStrategy,
+        process.platform === 'win32' ? WindowsSystemFileExecutionStrategy : SystemFileExecutionStrategy
```

Comment:
Just add two of them unconditionally, maybe we need to rename the old one to `UnixSystem...`. They should figure out if they're applicable.